tests/integration/compose/docker_compose_hdfs.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.3'
 services:
     hdfs1:
-        image: sequenceiq/hadoop-docker:2.7.0
+        image: prasanthj/docker-hadoop:2.6.0
```

Comment:
Could you please explain this?

Reply:
Old sequenceiq image has unsupported container image and we get warnings about deprecation. This was found here https://github.com/ClickHouse/ClickHouse/pull/66867 but not all tests work with new docker as we have changed network behaviour (different IPs)